0011 spider pitt housing opp

ben-nathanson opened the pull request:
Added pitt_housing_opp spider. Still having an issue with pytest regarding ABCs/Collections. See issue 39 on the bonfirefan/city-scrapers repo for details.

bonfirefan commented:
Same story here! Nerge in changes from `upstream/master` and commit those changes

bonfirefan commented:
Hey @ben-nathanson, this looks good. Did you mean to open a separate branch for `pa_development`? I see additional `pa_development` files here.

ben-nathanson commented:
Hi Bonnie! Thanks for reviewing my PR. I have added the new pitt_city_council spider and removed the erroneously included pa_development spider along with its associated test files. Let me know if that works. Thanks! -- Ben
